src/main/java/org/synyx/urlaubsverwaltung/security/oidc/OidcSecurityProperties.java

```diff
@@ -0,0 +1,68 @@
+package org.synyx.urlaubsverwaltung.security.oidc;
+
+import org.hibernate.validator.constraints.URL;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.constraints.NotEmpty;
+
+/**
+ * @author Florian Krupicka - krupicka@synyx.de
+ */
+@Validated
+@ConfigurationProperties("uv.security.oidc")
+public class OidcSecurityProperties {
```

Comment:
Eventuell eine Beispiel `application.properties` für oidc anbindung mit Erklärung?

Reply:
hm ne, wenn dann würde ich in der Properties datei Javadoc verwenden. Passt das?

Reply:
Wenn Javadoc, wie wäre es dann mit der Generierung von config metadata? Wird von allen großen IDEs unterstützt.

https://docs.spring.io/spring-boot/docs/current/reference/html/configuration-metadata.html
https://jakubstas.com/configuration-meta-data-in-spring-boot/

Reply:
ist alles schon da.